test_game_of_dbs.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+import implement_me
+import pytest
+import datasets
+from db_handler import recreate_all_tables
+from interface import Entry
+import ipaddr
+
+
+@pytest.fixture()
+def setup_data():
```

Comment:
This fixture is more of a setup function.
You could use a nice yield fixture here instead - first create everything, then yield, then drop.

Reply:
Right. Fixed